css.md

```diff
@@ -26,6 +26,19 @@ ITCSS organizes CSS into eight "groups" that can even be used to organize your L
 
 It's also a helpful pattern to prefix selectors (i.e. `.o-, .c-, .u-`) using these group names so that it's easy to identify the role of a selector in markup.
 
+### BEM
+
+Use the [BEM methodology](http://getbem.com/introduction/) to name class selectors.
+
+```html
+<div class="c-card c-card--theme">
```

Comment:
Simply for general reinforcement of semantic usage, I suggest setting this outer one to a `section`.

Reply:
not sure `section` makes sense out of context here

Reply:
`h3` is also out of context of a heading hierarchy, but for the sake of the example it serves the purpose. My point is to avoid defaulting to `div`, including in examples.

Reply:
not sure a card is a `section` is what I meant

Reply:
Ah. I'm not sure there's a better one. Maybe `article`, but not in most cases.

> The HTML <section> element represents a standalone section — which doesn't have a more specific semantic element to represent it — contained within an HTML document. Typically, but not always, sections have a heading. [source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

I'm in the camp of liberal use of `section`.

Reply:
Want me to use an article or media object example here for clarification?